README.md

```diff
@@ -695,6 +696,66 @@ OpenAI-compatible API. For example, the one spawned by `ilab model serve` or any
    ilab data generate --taxonomy-base=empty
    ```
 
+### 📇 Configure retrieval-augmented generation (developer preview)
+
+If you want to be able to use the developer preview of retrieval-augmented generation (RAG), you first need to enable developer preview features by setting `ILAB_FEATURE_SCOPE` to `DevPreviewNoUpgrade`, which tells InstructLab to enable developer preview features that might not be upgradable.  For many popular shells, this is done via the following:
+
+   ```shell
+   export ILAB_FEATURE_SCOPE=DevPreviewNoUpgrade
+   ```
+
+#### Converting documents
+
+Once you have enabled developer preview features, the first step is to convert your content into a structured form and then index that content for use in RAG by `ilab model chat`.  There are three options for doing the conversion:
+
+*Converting documents from a directory:* If you have all the source documents (e.g., PDF, Markdown) that you want to convert in a single local directory, you can convert them:
+
+   ```shell
+   ilab rag convert --input-directory=/my/source/documents/directory/goes/here/ --output-directory=/my/converted/documents/directory/goes/here/
+   ```
+
+We have tested this capability with PDF, Markdown, and Microsoft Word format documents.  Any of the [formats supported by Docling](https://ds4sd.github.io/docling/reference/document_converter/#docling.document_converter.InputFormat) should work.
```

Comment:
I don't see any other uses of the first person "we" in the rest of the README. Usually these things are in passive voice, no? e.g. "This capability has been tested with PDF and Markdown..."